docs/dev_guide/diagnostic_ref/dpct1038.rst

```diff
@@ -23,4 +23,64 @@ parameters. This may lead to incorrectly generated |dpcpp| code.
 Suggestions to Fix
 ------------------
 
-Review the kernel call inside the macro definition and adjust it manually, if needed.
\ No newline at end of file
+Review the kernel call inside the macro definition and adjust it manually, if needed.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    __global__ void kernel(int c, int d) {}
+    
+    void foo() {
+    #define KC(NAME, a, b, c, d) NAME<<<a, b, 0>>>(c, d);
+      KC(kernel, 3, 2, 1, 0)
+    }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+    void kernel(int c, int d) {}
+    
+    void foo() {
+    #define KC(NAME, a, b, c, d)                                                   \
+      dpct::get_default_queue().submit([&](sycl::handler &cgh) {                   \
+        auto c_ct0 = c;                                                            \
+        auto d_ct1 = d;                                                            \
+                                                                                   \
+        cgh.parallel_for(                                                          \
+            sycl::nd_range<3>(sycl::range<3>(1, 1, a) * sycl::range<3>(1, 1, b),   \
+                              sycl::range<3>(1, 1, b)),                            \
+            [=](sycl::nd_item<3> item_ct1) { kernel(c_ct0, d_ct1); });             \

```

Comment:
Please sync with Andy if SYCLomatic can be improved to process the line56.

Reply:
done